src/main/java/com/zebrunner/carina/webdriver/DriverHelper.java

```diff
@@ -496,6 +506,192 @@ public void addCookie(Cookie cookie) {
+        return getAllCookies().stream()
+                .map(cookie -> cookie.getName() + "=" + cookie.getValue() + ";")
+                .collect(Collectors.joining());
+    }
+
+    /**
+     * delete a specific cookie
+     *
+     * @param name the name of the cookie being deleted
+     */
+    public void deleteCookieNamed(String name) {
+        getDriver().manage().deleteCookieNamed(name);
+    }
+
+    /**
+     * delete all cookies in the driver session
+     */
+    public void deleteAllCookies() {
+        getDriver().manage().deleteAllCookies();
+    }
+
+    /**
+     * add an item to local storage
+     *
+     * @param name the name of the item to save
+     * @param value the value of the item to save
+     */
+    public void addToLocalStorage(String name, String value) {
+        ((JavascriptExecutor) getDriver()).executeScript(String.format("window.localStorage.setItem('%s','%s');", name, value));
+    }
+
+    /**
+     * add an item to local storage
+     *
+     * @param name the name of the item to save
+     * @param value the value of the item to save
+     */
+    public void addToLocalStorage(String name, Boolean value) {
+        addToLocalStorage(name, String.valueOf(value));
+    }
+
+    /**
+     * check to see if the item is present in local storage
+     *
+     * @param name the name of the item being stored
+     * @return boolean for if the item is present or not
+     */
+    public boolean isItemPresentInLocalStorage(String name) {
+        return ((JavascriptExecutor) getDriver()).executeScript(String.format("return window.localStorage.getItem('%s');", name)) != null;
+    }
+
+    /**
+     * gets the value of the stored item
+     *
+     * @param name the item that is stored
+     * @return value of the item
+     */
+    public String getValueFromLocalStorage(String name) {
+        return (String) ((JavascriptExecutor) getDriver()).executeScript(String.format("return window.localStorage.getItem('%s');", name));
+    }
+
+    /**
+     * gets the name of a stored item via an index
+     *
+     * @param index the position of where the item is stored
+     * @return name of the stored item
+     */
+    public String getNameFromLocalStorage(int index) {
+        return (String) ((JavascriptExecutor) getDriver()).executeScript(String.format("return window.localStorage.key('%s');", index));
+    }
+
+    /**
+     * remove an item from local storage
+     *
+     * @param name the name of the item to remove
+     */
+    public void removeFromLocalStorage(String name) {
+        ((JavascriptExecutor) getDriver()).executeScript(String.format("window.localStorage.removeItem('%s');", name));
+    }
+
+    /**
+     * clears the local storage
+     */
+    public void clearAllFromLocalStorage() {
+        ((JavascriptExecutor) getDriver()).executeScript("return window.localStorage.clear();");
+    }
+
+    /**
+     * gets the states of the redux store
+     * @return all store states
```

Comment:
Provide additional info or links about what is redux store and why we need methods to modify it

Reply:
I was told that we wanted to have support added for Redux (a JS framework) to Carina from @vdelendik . I don't have a ton of personal knowledge or experience about Redux myself.